main.py

```diff
@@ -1,5 +1,8 @@
-from gui.display_manager import DisplayManager
+from src.gui.display_manager import DisplayManager
+from mab.train import train
 
 if __name__ == '__main__':
-    display_manager = DisplayManager()
```

Comment:
but if I want to run a remote game, do I need to uncomment it?) can you leave the training locally?)

Reply:
Yes, the display manager code only should be run in main, not the training. This was left over from by accident or by a mistake.